src/pages/settings/Wallet/InternationalDepositAccount/substeps/Confirmation.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useCallback, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import CheckboxWithLabel from '@components/CheckboxWithLabel';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {CustomSubStepProps} from '@pages/settings/Wallet/InternationalDepositAccount/types';
+import * as BankAccounts from '@userActions/BankAccounts';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+const STEP_INDEXES = CONST.CORPAY_FIELDS.INDEXES.MAPPING;
+
+function TermsAndConditionsLabel() {
+    const {translate} = useLocalize();
+    return (
+        <Text>
+            {translate('common.iAcceptThe')}
+            <TextLink href={CONST.TERMS_URL}>{`${translate('common.addCardTermsOfService')}`}</TextLink>
+        </Text>
+    );
+}
+
+function Confirmation({onNext, onMove, formValues, fieldsMap}: CustomSubStepProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const [isSubmitting, setIsSubmitting] = useState(false);
+    const [corpayFields] = useOnyx(ONYXKEYS.CORPAY_FIELDS);
+
+    const getDataAndGoToNextStep = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.INTERNATIONAL_BANK_ACCOUNT_FORM>) => {
+        setIsSubmitting(true);
+        BankAccounts.createCorpayBankAccount(
+            {...formValues, ...values},
+            corpayFields?.classification ?? '',
+            corpayFields?.destinationCountry ?? '',
+            corpayFields?.preferredMethod ?? '',
+        ).then((response) => {
```

Comment:
Should we handle failure case? Currently, there are no feedback if the API failed

Reply:
I think we can display a error message in this case

Reply:
@shubham1206agra I mean we can handle errors in failureData and maybe display that error on the confirmation page

Reply:
There are no need any new component

Reply:
In createCorpayBankAccount function, let's add corpayFields.errors in failureData like this

https://github.com/Expensify/App/blob/3cc88f5a83a979c5e588e01065083a3d714e69da/src/libs/actions/Card.ts#L101

And then in Confirmation, let's add the error message to Form like here

https://github.com/Expensify/App/blob/3cc88f5a83a979c5e588e01065083a3d714e69da/src/pages/workspace/expensifyCard/issueNew/ConfirmationStep.tsx#L129